src/SystemCommands-RefactoringSupport/RBPullUpPreview.class.st

```diff
@@ -0,0 +1,90 @@
+"
+I am a preview of Pull up refactoring, I can select the superClass of pull up methos, and select the methods of class to pull up
+"
+Class {
```

Comment:
Can you write example what UI will allow to do?
Currently in the browser pushUp command allow to moved methods of selected class to superclass.
Here will user select the superclass where to move methods?